packages/next-swc/crates/next-core/src/next_app/app_page_entry.rs

```diff
@@ -190,10 +193,13 @@ async fn wrap_edge_page(
         INNER.to_string() => entry
     };
 
-    let wrapped = context.process(
-        Vc::upcast(source),
-        Value::new(ReferenceType::Internal(Vc::cell(inner_assets))),
-    );
+    let wrapped = context
+        .process(
+            Vc::upcast(source),
+            Value::new(ReferenceType::Internal(Vc::cell(inner_assets))),
+        )
+        .await?
```

Comment:
`await?.context("...")` looks bad, because we return one `Err` in `Result<Result<T, E>, E>` without context

Reply:
But it's not specific to this PR; I just found it